cmd/govim/testdata/show_message.txt

```diff
@@ -0,0 +1,19 @@
+# Test that popup shows up when message is of Error type
+
+[!vim] [!gvim] skip
+
+# For some reason, the new-style popup test fails on gvim.
+# https://github.com/govim/govim/issues/351
+[!vim:v8.1.1649] skip
```

Comment:
In https://github.com/govim/govim/pull/501 we increased the minimum version of Vim required to run `govim` to v8.1.1711, so conditions like this are now redundant.

I've just raised https://github.com/govim/govim/issues/518 to tidy this up everywhere else.